### PR TITLE
[Shadow] Update guide view recommendations to be more accurate

### PR DESCRIPTION
**Pull request by DoxAshe**
### Description

Some of the suggestions in the guide view had some mistakes in them.  I have fixed them to remove any inaccurate suggestions, and added in some additional information that is useful for shadow priest's current rotation.  
I prioritized these issues to make sure all information in the guide is accurate for the current rotation.  I hope to soon add statistics and suggestions for Shadow's Hero Talents.  

- Test report URL: `report/t7HMBVqDFa9PxgcL/16-Heroic+Sikran,+Captain+of+the+Sureki+-+Kill+(3:02)/Kiyourah/standard/statistics`
- Screenshot(s):

### CODE REVIEW

**Comment by DoxAshe** [comment]
I'm not sure why the e2e tests are failing, I can't seem to figure out how to fix the issue.

**Comment by ToppleTheNun** [comment]
The issue is because GitHub updated the Ubuntu runners to support Ubuntu 24.04, which Playwright doesn't support atm: microsoft/playwright#30368

**Comment by ToppleTheNun** [comment]
Should be fixed in `the-war-within`.

File: src/analysis/retail/priest/shadow/modules/talents/UnfurlingDarkness.tsx [hunk 1]
```diff
@@ -76,21 +81,28 @@ class UnfurlingDarkness extends Analyzer {
...
         procs.
       </p>
     );
 
     const data = (
       <div>
-        <strong>Unfurling Darkness breakdown</strong>
+        <strong>Unfurling Darkness Usage</strong>
         <GradiatedPerformanceBar good={goodUD} bad={badUD} />
+        <strong>Unfurling Darkness Procs</strong>
```
Comment (trevorm4) [inline src/analysis/retail/priest/shadow/modules/talents/UnfurlingDarkness.tsx:104]: Can use spellLink's if you want

**Review by trevorm4 — APPROVED** [review]

**Pull request merged**